src/formats.c

```diff
@@ -1464,42 +1465,46 @@ static void test_fmt_split_unifies_case_3(struct fmt_main *format,
 	// Lower case
 	strlwr(cipher_copy + index);
 
-	if (strcmp(cipher_copy, ciphertext))
-	if (format->methods.valid(cipher_copy, format)) {
+	if (strcmp(cipher_copy, ciphertext)) {
+		if (format->methods.valid(cipher_copy, format)) {
```

Comment:
May be @magnumripper didn't indent properly, to make merging core changes easier, I don't know.

Also, I wouldn't mix format.c changes and format specific changes into one commit.
Instead, the formats.c changes should come first, format specific changes in a separate commit.

Reply:
formats.c changes can not come first.  they will not pass bot checks without the changes to the format files.

Reply:
This code is not part of core.  Core has no concept of --test-full=x so there should be no merge conflict

Reply:
No this is Jumbo-only, originating from some GSoC task. Perhaps the original intention was to avoid indenting so much.

I agree that ideally this should be separate commits (although they can very well be in one same PR). Probably not important enough to require a rebase though.